test/test_special_elixirs.jl

```diff
@@ -109,7 +109,7 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
     end
 
     @timed_testset "MHD" begin
-      trixi_include(@__MODULE__, joinpath(EXAMPLES_DIR, "tree_2d_dgsem", "elixir_mhd_alfven_wave_mortar.jl"),
+      trixi_include(@__MODULE__, joinpath(EXAMPLES_DIR, "tree_2d_dgsem", "elixir_mhd_alfven_wave.jl"),
```

Comment:
Out of curiosity: why change?

Reply:
Because it has less DOF and is thus faster for CI.

Reply:
Ok. I thought that whoever put it there might have had a reason for using the example with non-conforming cells (e.g., to specifically test AD with mortars). But if that isn't the case, feel free to resolve the conversation.

Reply:
I put it there because I got a weird error when running that elixir in the REPL. After quite some time, I discovered that the problem was just that my computer didn't have enough memory to store the huge dense matrix we used to compute all eigenvalues :see_no_evil: I just left it there with reduced resolution. Thus, from the point of view of the guy who put this test there, there is no need for the mortars here :sweat_smile: